.golangci.yml

```diff
@@ -106,6 +105,7 @@ linters-settings:
       - valSwap
       - wrapperFunc
       - yodaStyleExpr
+      # - ifElseChain
```

Comment:
Any reason we commented this one out when it was uncommented before?

Reply:
It reports to refactor else-if chains into switch statements, like:
https://github.com/kubernetes/release/blob/af8e773465759f580439c253259db258a56199f6/pkg/git/git.go#L111-L122

I'm not sure if we should do this and I tend to say no. But I'm open to go with the defaults.

Reply:
Let's leave that one commented out for now.
I'd say once we hit three or more chains and there's a common pattern, maybe consider `switch` statements, but I don't think we need to nit on it at the moment.